.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: java
+dist: trusty
+sudo: true
+before_cache:
+  - rm -f $HOME/.gradle/caches/modules-2/modules-2.lock
```

Comment:
I certainly hope not or it would be a horrible security failing.  You'll have to encrypt it on the command line using the travis cli.  I can do it / show you how if you want.